virttest/qemu_devices/qcontainer.py

```diff
@@ -1499,7 +1499,8 @@ def usbc_by_params(self, usb_name, params, pci_bus={"aobject": "pci.0"}):
 
     # USB Device related methods
     def usb_by_variables(self, usb_name, usb_type, controller_type, bus=None,
-                         port=None, serial=None):
+                         port=None, serial=None, hostbus=None, hostaddr=None,
+                         hostport=None, vendorid=None, productid=None):
```

Comment:
Suggest packing the device specified into one variable, e.g. `dev_options`

Reply:
Updated, thanks!